site/docs/components/button-group.md

```diff
@@ -5,40 +5,22 @@ description: Group a series of buttons together on a single line with the button
 group: components
 toc: true
 ---
+Styling options for button groups, checkout button groups in web-app storybook to add to the site.
```

Comment:
the link to storybook? https://flosports-webapp-storybook.stag.flokubernetes.com

Reply:
Thanks -  Added